src/components/Contacts/ContactsHeader/ContactsHeader.tsx

```diff
@@ -1,10 +1,192 @@
-import { Box, colors } from '@material-ui/core';
-import React from 'react';
+import { Box, styled, IconButton, Hidden } from '@material-ui/core';
+import { FilterList, FormatListBulleted, ViewColumn } from '@material-ui/icons';
+import { ToggleButton, ToggleButtonProps } from '@material-ui/lab';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import theme from '../../../theme';
+import { CheckBox, CheckBoxState } from '../CheckBox/CheckBox';
+import { StarContactIcon } from '../StarContactIcon/StarContactIcon';
+
+interface Props {
+  activeFilters: boolean;
+  filterPanelOpen: boolean;
+  toggleFilterPanel: () => void;
+  totalContacts?: number;
+}
+
+enum ContactsTableDisplayState {
+  'list',
+  'columns',
+}
+
+const HeaderWrap = styled(Box)(({}) => ({
+  height: 96,
+  padding: theme.spacing(1),
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'space-evenly',
+  alignItems: 'center',
+  backgroundColor: theme.palette.background.default,
+}));
+const FilterButton = styled(IconButton)(
+  ({
+    activeFilters,
+    panelOpen,
+  }: {
+    activeFilters: boolean;
+    panelOpen: boolean;
+  }) => ({
+    display: 'inline-block',
+    width: 48,
+    height: 48,
+    borderRadius: 24,
+    margin: theme.spacing(1),
+    backgroundColor: activeFilters
+      ? '#FFCF07'
```

Comment:
Let's get this in the theme. I'll try and get John's PR merged today which has it https://github.com/CruGlobal/mpdx-react/pull/49/files#diff-0e21fb1c9519a397050defc2148a2d1b966a0982cc4cf196e1e6b007f8d095e5R7